### PR TITLE
feat: add InRandomOrder method

### PR DESCRIPTION
**Pull request by Kamandlou**
## 📑 Description
add InRandomOrder method

## ✅ Checks

- [x] My pull request adheres to the code style of this project
- [x] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [x] All the tests have passed

### CODE REVIEW

File: database/gorm/query.go [hunk 1]
```diff
@@ -455,6 +455,25 @@ func (r *QueryImpl) OrderByDesc(column string) ormcontract.Query {
 	return r.Order(fmt.Sprintf("%s DESC", column))
 }
 
+func (r *QueryImpl) InRandomOrder() ormcontract.Query {
+	order := ""
+	switch r.Driver() {
+	case "mysql":
```
Comment (hwbrzzl) [inline database/gorm/query.go:461]: S: Use `orm.DriverMysql.String()` instead.

File: database/gorm/query_test.go [hunk 2]
```diff
@@ -1973,6 +1973,24 @@ func (s *QueryTestSuite) TestOrderByDesc() {
...
+			s.True(user.ID > 0)
+
+			user1 := User{Name: "random_order_user", Avatar: "random_order_avatar1"}
+			s.Nil(query.Create(&user1))
+			s.True(user1.ID > 0)
+
+			var users []User
+			s.Nil(query.Where("name = ?", "random_order_user").InRandomOrder().Get(&users))
+			s.True(len(users) == 2)
+		})
```
Comment (hwbrzzl) [inline database/gorm/query_test.go:1990]: S: This case can't confirm the validity of `InRandomOrder`, we can create multiple users(for example 10 times), query them second times, then compare the ID of index 0, 5, 10.

**Review by hwbrzzl — CHANGES_REQUESTED** [review]

File: database/gorm/query.go (hunk 1 above)
Comment (Kamandlou) [reply database/gorm/query.go]: when I use Postgresql database `r.Driver()` returns "postgres" but `ormcontract.DriverPostgresql` returns "postgresql" and it causes a problem

Comment (Kamandlou) [reply database/gorm/query.go]: what should I do in this case?
I can define a default value for `order` variable. 
like this: `order := "RAND()"`

Comment (hwbrzzl) [reply database/gorm/query.go]: We can define a new constant: `ormcontract.DriverPostgres`, we want to use `postgres` in the future.

File: contracts/database/orm/constants.go [hunk 3]
```diff
@@ -3,6 +3,7 @@ package orm
 const (
 	DriverMysql      Driver = "mysql"
 	DriverPostgresql Driver = "postgresql"
```
Comment (hwbrzzl) [inline contracts/database/orm/constants.go:5]: Could you please add the DEPRECATED tag for this line?

Comment (Kamandlou) [reply contracts/database/orm/constants.go:5]: Yes

**Review by hwbrzzl — APPROVED** [review]
Great work!

**Pull request merged**